src/quacc/settings.py

```diff
@@ -121,6 +122,42 @@ class QuaccSettings(BaseSettings):
         ),
     )
 
+    # ---------------------------
+    # ESPRESSO Settings
+    # ---------------------------
+    ESPRESSO_CMD: Path = Field(
+        Path("pw.x"),
+        description=("Path to the espresso executable."),
+    )
+    ESPRESSO_PP_PATH: Optional[Path] = Field(
+        None,
+        description=("Path to a pseudopotential library for espresso."),
+    )
+    ESPRESSO_PRESET_DIR: Path = Field(
+        resources.files(espresso_presets),
+        description="Path to the espresso preset directory",
+    )
+    ESPRESSO_PH_CMD: Path = Field(
+        Path("ph.x"),
+        description=("Path to the espresso ph.x executable."),
+    )
+    ESPRESSO_NEB_CMD: Path = Field(
+        Path("neb.x"),
+        description=("Path to the espresso neb.x executable."),
+    )
+    ESPRESSO_Q2R_CMD: Path = Field(
+        Path("q2r.x"),
+        description=("Path to the espresso q2r.x executable."),
+    )
+    ESPRESSO_MATDYN_CMD: Path = Field(
+        Path("matdyn.x"),
+        description=("Path to the espresso matdyn.x executable."),
+    )
+    ESPRESSO_DYNMAT_CMD: Path = Field(
+        Path("dynmat.x"),
+        description=("Path to the espresso dynmat.x executable."),
+    )
+
```

Comment:
Are these executables typically found within a single directory? If so, perhaps it would be easier to have a single variable (e.g. `ESPRESSO_EXE_DIR`). That would help later since if there are a lot more executables one might want to add, they won't have to enumerate each one as a setting. However, if that's not the typical structure of QE's executables, then this comment isn't relevant. (As you can tell, I have not used QE before).